Document/0x05b-Basic-Security_Testing.md

```diff
@@ -812,7 +989,7 @@ There are several ways to bypass certificate pinning for a black box test, depen
 
 For most applications, certificate pinning can be bypassed within seconds, but only if the app uses the API functions that are covered for these tools. If the app is implementing SSL Pinning with a custom framework or library, the SSL Pinning must be manually patched and deactivated, which can be time-consuming.
 
-##### Bypass Custom Certificate Pinning Statically
+###### Bypass Custom Certificate Pinning Statically
 
```

Comment:
We should shift everything to the SSL Pinning test case and merge it together. If we describe SSL Pinning here, then we would also need to describe other things such as Root Detection, bypassing anti-debugging, code obfuscation etc. which should all be described in the relevant chapters.

Reply:
I agree :)